tests/render/test_render_BulletListContentBlock.py

```diff
@@ -76,15 +76,27 @@ def test_all_expectations_using_test_definitions():
         len(test_files) == 61
     ), "Something went wrong when collecting JSON Expectation test fixtures"
 
+    # Not ported over to V1 so ignore for purposes of this test
```

Comment:
Should we delete this expectations for V1?

Reply:
Probably? Not sure what the plan was here to be honest

Reply:
I looked at [this spreadsheet](https://docs.google.com/spreadsheets/d/1aIIelGPVdHbijKsqNsTlheg9vAAayi3t_OYHwT9og_o/edit#gid=0) to get a list of the expectations we are going to support. Oh these 4, I see `expect_column_pair_values_A_to_be_greater_than_B` to be on that list. I also see it on your [spreadsheet in row 14](https://docs.google.com/spreadsheets/d/1roV3afWMOzpU52p_dQyab_t64js9aic0ev4eNdzBwEI/edit#gid=0).

I think we should delete the top 3 (I'll file a ticket). The last one should be ported over so should be passing?